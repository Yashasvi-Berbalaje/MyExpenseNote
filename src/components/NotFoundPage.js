import {Link} from 'react-router-dom';
import React from 'react';


  const NotFoundPage= ()=>(
    <div>
      Error-404 not found!!!!!
      <Link to='/Dashboard'>
       Go Home
      </Link>
  
    </div>
  );
  export default NotFoundPage;