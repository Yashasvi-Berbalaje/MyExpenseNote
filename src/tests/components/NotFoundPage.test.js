import React from 'react';
import {shallow} from 'enzyme';
import NotFoundPage from '../../components/NotFoundPage';
import toJSON from 'enzyme-to-json';

test('Test case for Not Found Page',()=>{
    const action= shallow(<NotFoundPage/>);
    expect(toJSON(action)).toMatchSnapshot();
})