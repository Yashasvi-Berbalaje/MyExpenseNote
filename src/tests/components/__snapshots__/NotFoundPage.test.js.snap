// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test case for Not Found Page 1`] = `
<div>
  Error-404 not found!!!!!
  <Link
    to="/Dashboard"
  >
    Go Home
  </Link>
</div>
`;
